### DISCUSSION TITLE
How to set the style of links in the body

### DISCUSSION THREAD

**Discussion by allen0234** [web-infra-dev/rspress#535] [Q&A]
想把正文里面链接的字体颜色设置为默认主题色，而不是鼠标悬浮的时候才显示为主题色，不知道怎么设置。目前rspress文档里面自定义样式的指导内容还是比较少。

**Comment by sanyuan0704** [comment]
通过 globalStyles 配置往 .rspress-doc 下面加样式就可以

**Reply by allen0234** [reply]
颜色设置成功了。不过同时设置text-decoration为outline的话，跟自带的虚线下划线会同时出现。
.rspress-doc a {
    color: var(--rp-c-brand);
    
}

**Reply by allen0234** [reply]
@sanyuan0704
